debian/control

```diff
@@ -375,6 +375,7 @@ Package: ceph-osd
 Architecture: linux-any
 Depends: ceph-base (= ${binary:Version}),
          lvm2,
+         python-pkg-resources,
```

Comment:
i don't think we need this. as `dh_python2 -p ceph-osd` and `${python:Depends}` sub variable should have taken care of this. and i have the same question from Alfredo.

Reply:
Before going into any details on these changes, I would love to clarify my questions above:

* In what circumstances this happens? (we've never seen it anywhere)
* What version/distro is this?
* How can I reproduce the problem?

Reply:
Agreed `${python:Depends}` should have found this one.